mongo_database.py

```diff
@@ -0,0 +1,186 @@
+from mongoengine import *
+from pprint import pprint
+from math import ceil
+from datetime import datetime
+
+
+class Target(EmbeddedDocument):
+    killTarget = StringField(required=True)
+    healTarget = StringField(required=True)
+    checkTarget = StringField(required=True)
```

Comment:
not sure how this works with all required, but i assume you tested that it works

Reply:
> I couldn't get your branch to run on my machine, but the code in this PR makes sense.

you need to set up a mongodb instance on your local machine for this to work
I also added deployment testing using Docker
Let me know if you want to get it working on your end, it's a little complicated haha

Reply:
there will always be a value in those fields so it should be ok